qt/aqt/import_export/import_csv_dialog.py

```diff
@@ -52,13 +49,9 @@ def _setup_ui(self, path: str) -> None:
         self.setWindowTitle(tr.decks_import_file())
 
     def reject(self) -> None:
+        if self.mw.col:
+            self.mw.col.set_wants_abort()
```

Comment:
Hmm, I think this might still be problematic: consider the case where the user has left the importing window open, then later closes it while some other operation is running. Would it make sense to only call this if the window is still modal?

Reply:
Calling it only when the window is modal sounds reasonable.